CDP4Common/Exceptions/Cdp4ModelValidationException.cs

```diff
@@ -2,7 +2,7 @@
 // <copyright file="Cdp4ModelValidationException.cs" company="RHEA System S.A.">
 //    Copyright (c) 2015-2020 RHEA System S.A.
 //
-//    Author: Sam Gerené, Merlin Bieze, Alex Vorobiev, Naron Phou
+//    Author: Sam Gerené, Merlin Bieze, Alex Vorobiev, Naron Phou, Aelxander van Delft, Nathananiel Smiechowski
```

Comment:
Typo in my name. Cannot agree with that ;) 
Nathanael instead of Nathaniel.

Reply:
oops, will fix that!